### DISCUSSION TITLE
bulk import custom field multiobject

### DISCUSSION THREAD

**Discussion by uRhaineWork** [netbox-community/netbox#9449] [Q&A]
Hi, I would like to bulk import data with custom field multiobject type pulling dcim.locations data. When I try to import i get error:
Row 1 cf_custom_location: Enter a list of values. 
I'm not clear on the programming side, which type of string should I use. I understand that I cant enter value eg. 'ExampleLocation1' which i "see", only some ID number or some programming string? I tried reading some topcis about it, but I'm not sure if they cover my example eg. https://github.com/netbox-community/netbox/issues/1792 , as there are no clear examples anywhere.

**Comment by robbagithub** [comment]
Hi, 

I've got the same issue, did you find out what the solution was?

Cheers,
Rob.

**Reply by kkthxbye-code** [reply]
I believe it has just never worked, there was some missing logic to allow input of multiple values.

Please see: https://github.com/netbox-community/netbox/pull/11712

When the fix gets released, you should be able to import it like "32133,5432234,1232", that being the ID's of the referenced objects, enclosed in `"` and separate by `,`.

**Comment by robbagithub** [comment]
Thanks for the response. We tried several options, then removed the multi object field for bulk import. We'll change the datatype in a later release hopefully.
